container/Containerfile

```diff
@@ -0,0 +1,20 @@
+# Build stage
+
+FROM alpine AS build
+RUN apk add --no-cache --update build-base linux-headers python3 python3-dev py3-pip
+WORKDIR /build
+COPY ovirt-imageio.tar.gz .
+RUN python3 -m venv /venv
+RUN /venv/bin/pip install ovirt-imageio.tar.gz
+
+# Final stage
+
+FROM alpine
+RUN apk add python3
+RUN addgroup --system ovirtimg
+RUN adduser --system --ingroup ovirtimg ovirtimg
+RUN mkdir /run/ovirt-imageio /etc/ovirt-imageio
```

Comment:
We don't need to create /run/ovirt-imageio since we can disable the control service
(if you rebase on #146).